doc/Learn/Tutorials/Markup/HowTo-CustomMarkupProject-Theme.md

```diff
@@ -0,0 +1,312 @@
+---
+uid: Learn.Tutorials.HowToCustomMarkupProjectTheme
+---
+
+# C# Markup project setup
+
+In this tutorial you'll learn how to set up an Uno Platform project to use C# Markup to change the Theme. 
+
+1. For this tutorial we will continue to use the Sample created on the [Custom your own C# Markup - Learn how to use Toolkit](xref:Learn.Tutorials.HowToCustomMarkupProjectToolkit) tutorial and changed on the [Custom your own C# Markup - Learn how to change Visual States and User Controls](xref:Learn.Tutorials.HowToCustomMarkupProjectVisualStates).
+
+# Themes Overview
+
+### Uno [Themes](https://platform.uno/docs/articles/external/uno.themes/doc/themes-overview.html) Styles
+
+[Fluent](https://platform.uno/docs/articles/external/uno.themes/doc/fluent-getting-started.html) Controls Styles are the **Default Theme** for Uno Platform.
+Uno Platform 3.0 and above supports control styles conforming to the Fluent design system.
+
+[Uno.Themes](https://github.com/unoplatform/Uno.Themes) is the repository for add-ons NuGet packages that can be added to any new or existing Uno solution.
+
+It contains two libraries
+
+- [Material Overview](https://platform.uno/docs/articles/external/uno.themes/doc/material-getting-started.html)
+
+- [Cupertino Overview](https://platform.uno/docs/articles/external/uno.themes/doc/cupertino-getting-started.html)
+
+To help you style your application with a few lines of code to include and change:
+
+- Color system for both Light and Dark theme
+- Styles for existing WinUI controls like Buttons, TextBox, etc.
+- Change the Font of the Application
+
+## Quick Start
+
+In the two links above, you have a Getting Started for Material and Cupertino. 
+
+But the difference between they are the references that can be included on the Solution.
+
+You may notice that there are only minor differences in the XAML configuration in the link above and in this current tutorial.
+
+Since we are using the Default Fluent Theme in the current project, we can choose to stay with it or add a different one.
+
+Below we can see how to include and how to customize each of the two.
```

Comment:
```suggestion
## Quick Start

In the two links above, you have a Getting Started for Material and Cupertino. 

But the difference between they are the references that can be included in the Solution.

You may notice that there are only minor differences in the XAML configuration in the link above and in this current tutorial.

Since we are using the Default Fluent Theme in the current project, we can choose to stay with it or add a different one.

Below we can see how to include and how to customize each of the two.
```